### ISSUE TITLE
Selection of Responses

### ISSUE THREAD

**Issue by marksherman** [ec-idds/programmer-quiz#35]
- [ ] when a response is selected, change color or something to indicate its selection
- [ ] selection should be programmatically available so it can be saved when the "next" button is pressed
- [ ] save on "next" button to local storage

**Comment by marksherman** [comment]
Looks like it depends on #16  which may or may not actually be a separate issue
